inc/content-feed.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * The template used for displaying a feed of posts
+ *
+ * @package MIT_Libraries_Child
+ * @since Twenty Twelve 1.0
```

Comment:
If this is a brand new theme file, the "since" tag is meant to be used to indicate when the file was added to the theme - so in this case either 2.2.1 (or more accurately 2.2.2, since I'm hoping to bump the theme version before deploying)